exampleCourse/questions/template/multiple-choice/none-of-the-above/question.html

```diff
@@ -0,0 +1,10 @@
+<pl-question-panel>
+  <p>Which of the following is <strong>not</strong> a common sorting algorithm taught in introductory Computer Science courses?</p>
+</pl-question-panel>
+
+<pl-multiple-choice answers-name="sorting">
+  <pl-answer correct="false">Bubble Sort</pl-answer>
+  <pl-answer correct="false">Merge Sort</pl-answer>
+  <pl-answer correct="false">Quick Sort</pl-answer>
+  <pl-answer correct="true">None of the above</pl-answer>
```

Comment:
Can we avoid double negatives here? I'm thinking about using incorrect values as the options, e.g. "squiggle sort", "hash sort", "quantum sort".

Reply:
Changed here https://github.com/PrairieLearn/PrairieLearn/pull/12139/commits/bee5004c81e217755cadcb61821870526805bd9f